data/en/imageformats.json

```diff
@@ -4,11 +4,19 @@
 	"syntax": "imageFormats()",
 	"returns": "struct",
 	"related": [],
-	"description": "",
+	"description": "Returns all supported image formats that can be read and written.",
```

Comment:
Could you assign `encoder`/`decoder` to read/write because this are the keys, the output structure uses?

Reply:
Done in https://github.com/foundeo/cfdocs/pull/870/commits/09111c0648dea63fb08b6cbe127d90ebf8417ec5.